setup.py

```diff
@@ -43,12 +43,11 @@
 
 ydata_requires = [
     # preferably install using make install-ydata
-    'ydata-synthetic>=0.3.0,<0.4',
+    'ydata-synthetic==0.6.1',
```

Comment:
We are now pinning them, and updating would be done upon our review of their update.

Reply:
Let's add in documentation, if one wants to update to the updated version of ``ydata`` we would welcome such contribution, and we can review.